### PR TITLE
accurate coronaTestDates

### PR DESCRIPTION
**Pull request by y-golde**
## The issue
dates are being sent from the API in ISO format (e.g. "2020-11-19T20:00:00.000Z"), those dates are later being formatted to a Date object by JS's new Date() function,
JS's Date() constructor **rounds** those datetimes to dates so "2020-11-19T20:00:00.000Z" now becomes "2020-11-20T00:00:00.000Z" which is displayed in the entirety of the project ( 😱 )
## The solution
instead of rounding , I've added a pre-emptive floor to the date (by using .split('T')[0]) and then formatting the date, and added a ceil to the search (so in this example searching from  2020-11-20T00:00:00.000Z instead of  2020-11-19T00:00:00.000Z)

fixes [1340](https://dev.azure.com/spectrumFactory/CoronaI/_workitems/edit/1340/)

### CODE REVIEW

**Review by OrelKaravani — DISMISSED** [review]

File: client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts [hunk 1]
```diff
@@ -395,7 +395,7 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
     }
 
     const getFormattedDate = (date: string) => {
-        return format(new Date(date), 'dd/MM')
+        return format(new Date(date.split('T')[0]), 'dd/MM')
```
Comment (MayaSasson) [inline client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts:398]: maybe 'T' can be a const? can explain it better

Comment (MayaSasson) [reply client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts:398]: and maybe export new Date(date.split('T')[0]) to a function

Comment (y-golde) [reply client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts]: seperated to more verbose `truncateDate` [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1124/commits/c531470dd074f440425d0062302e18bde3deb950)

File: client/src/Utils/DateUtils/formatDate.ts [hunk 2]
```diff
@@ -16,4 +16,8 @@ export const formatDateTime = (date: Date): string => {
     ) 
 }
 
+export const truncateDate = (dateInISO : string): Date => {
+    return new Date(dateInISO.split('T')[0])
```
Comment (ItayBenMoshe) [inline client/src/Utils/DateUtils/formatDate.ts:20]: maybe export T as a const

**Review by MayaSasson — DISMISSED** [review]

File: client/src/Utils/DateUtils/formatDate.ts (hunk 2 above)
Comment (y-golde) [reply client/src/Utils/DateUtils/formatDate.ts]: exported [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1124/commits/0127af30717f1f267fcf71909a7b5d59349f07dc)

**Review by ItayBenMoshe — APPROVED** [review]

**Review by MayaSasson — APPROVED** [review]

**Pull request merged**